examples/hello-world/format-example.cpp

```diff
@@ -0,0 +1,130 @@
+
+#include "internalGroup1.h"
+#include "slang.h"
+#include <algorithm>
+
+#include "includeGroup2.h"
+#include <vector>
+
+#include <vulkan.h>
+
+#ifdef __linux__
+#include <X11/Xlib.h>
+#else
+#include <Windows.h>
+#endif
+
+namespace slang
+{
+    /// <summary>
+    /// Summary of TestClass
+    /// </summary>
+    class TestClass
+    {
+    private:
+        int examplePrivateMember; // Trailing comment1
+        int* examplePointerMember = nullptr; // trailing coment2
```

Comment:
I am not a fan of AlignTrailingComments for the same reason to keep changes local. When you change the variable name of one line, you don't want to cause nearby lines to change because of that.